vin/app/controllers/vin/sub_controller.rb

```diff
@@ -1,87 +0,0 @@
-class Vin::SubController < ApplicationController
-respond_to :json
-  
-  # POST /vin/sub
-  def create
-    @address = Address.create(address_params)
-
-    @subscriber = Subscriber.new(subs_params)
-    
-    @subscriber.address = @address
-
-    already = !@subscriber.save
-
-    if already
-      render json: { errors: ["Subscriber already registered!"]}
```

Comment:
This is a very rudimentary way of handling errors. The record could have not been saved for multiple reasons